.changelog/unreleased/bug-fixes/771-kvindexer-parsing-big-ints.md

```diff
@@ -0,0 +1,2 @@
+- `[state/kvindex]` Querying event attributes that are bigger than int64 is now enabled. We are not supporting reading floats from the db into the indexer nor parsing them into BigFloats to not introduce breaking changes in minor releases.
+  ([\#771](https://github.com/cometbft/cometbft/pull/771))
```

Comment:
> Querying event attributes that are bigger than int64 is now enabled.

This makes sense to me as something to communicate to users.

> We are not supporting reading floats from the db into the indexer nor parsing them into BigFloats to not introduce breaking changes in minor releases.

What does this, however, mean for users?

Reply:
Maybe this does not have to be in the changelog, but only in the docs. This is basically the same behaviour as they had before, but they might have been unaware of this. I agree that maybe this should be removed frm the changelog but left in the docs.